app/src/main/java/com/nilhcem/blenamebadge/ui/message/MessageActivity.kt

```diff
@@ -331,13 +310,39 @@ class MessageActivity : AppCompatActivity() {
         builder.create().show()
     }
 
-    private fun convertToDeviceDataModel(): DataToSend {
-        return DataToSend(listOf(Message(content.text.trim().toString(), flash.isChecked, marquee.isChecked, Speed.values()[speed.selectedItemPosition], Mode.values()[mode.selectedItemPosition])), invertLED.isChecked)
+    private fun convertTextToDeviceDataModel(): DataToSend {
+        return DataToSend(
+            listOf(
```

Comment:
Move to previous line

Reply:
yes sir, I am learning the type of indentation required now. just a min

Reply:
made the changes, have a look